pkg/ui/console/client.go

```diff
@@ -152,7 +153,7 @@ func (c Client) DownloadPolicy(ctx context.Context, args []string) error {
 func (c Client) RunPolicies(ctx context.Context, args []string, policyName, outputDir, subPath string, stopOnFailure, skipVersioning, failOnViolation, noResults bool) error {
 	c.c.Logger.Debug("Received params: args: %v, policyName: %s, outputDir: %s, stopOnFailure: %v, skipVersioning: %v, failOnViolation: %v, noResults: %v", args, policyName, outputDir, stopOnFailure, skipVersioning, failOnViolation, noResults)
 
-	policiesToRun, err := client.FilterPolicies(args, c.cfg.Policies, policyName, subPath)
+	policiesToRun, err := policy.FilterPolicies(args, c.cfg.Policies, policyName, subPath)
```

Comment:
Move to the other PR I suppose

Reply:
I actually after going further into that these functions belong to the console, since its console argument parsing its only used here. Should move to `ui/console/policy.go` I will refactor that after I merge this PR and #322